lib/service_actor/result.rb

```diff
@@ -109,6 +117,10 @@ def deconstruct_keys(keys)
 
   attr_reader :data
 
+  def convert_key(key)
+    key.to_sym
```

Comment:
I’d like to avoid adding too many methods to `Result` since that means that it’s an input name that can’t be used anymore.

Can you change that to calling `to_sym` directly in every method? 🙏🏻